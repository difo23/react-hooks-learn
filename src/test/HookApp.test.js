import React from 'react';
import { shallow } from 'enzyme';
import { HookApp } from '../HookApp';



describe('Prueba en <HookAp>', () => {
    test('should show successful', () => {

        //Prueba
        const wrapper = shallow(<HookApp />);

        // Crea carpeta snapshot en test
        expect(wrapper).toMatchSnapshot();

    })


})
