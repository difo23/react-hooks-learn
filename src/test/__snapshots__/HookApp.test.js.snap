// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba en <HookAp> should show successful 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
